content/docs/components/accordion/theming.mdx

```diff
@@ -2,3 +2,188 @@
 id: accordion
 scope: theming
 ---
+
+The `Accordion` component is a multipart component. The styling needs to be
+applied to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
```

Comment:
I've prepared an anatomy illustration for you. Grab [this code](https://gist.github.com/nikolovlazar/bec580e8d8d2eb12bed7c7aa6086275a) and add the following section as the first:

```markdown
## Anatomy

- A: `root`
- B: `container`
- C: `button`
- D: `panel`
- E: `icon`

[PLACE THE ILLUSTRATION CODE HERE]
```